src/content/blog/fastify.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: "Revamping our API pt. 1: Moving from express to Fastify"
```

Comment:
The title is a bit wordy. How about:

- Migrating from Express to Fastify
- Migrating from ExpressJS - Part 1